tests/many_to_one/tests.py

```diff
@@ -510,6 +510,14 @@ def test_fk_assignment_and_related_object_cache(self):
         with self.assertRaisesMessage(ValueError, msg):
             ToFieldChild.objects.create(parent=p)
 
+        # Create parent and child, save parent, save child, parent_id should be set
```

Comment:
A separate test method is fine. The existing super long test is probably a docstring test that was never cleaned up.